figures/templates/figures/admin_dropdown_filter.html

```diff
@@ -0,0 +1,28 @@
+{# Attribution: Original code here: https://github.com/mrts/django-admin-list-filter-dropdown #}
```

Comment:
It's a lot safer to use `load` tags before comments. Both Python `import` and Django `load` are "big deal" and usually the order matters, esp. when using something like `from __future__ import unicode_literals`.

Therefore, I think it's just proper hygiene to use `load` first.

```suggestion
{% load i18n %}

{# Attribution: Original code here: https://github.com/mrts/django-admin-list-filter-dropdown #}
```